### DISCUSSION TITLE
Link preview

### DISCUSSION THREAD

**Discussion by imdevarsh** [ScratchAddons/ScratchAddons#3842] [Ideas]
Hi there, I have a new idea for the Scratch Addons project, it's Link previews!
For example, let's say I want to go to a profile. Somebody gives me a link https://scratch.mit.edu/users/god286 so it would show something similar to this when I hover over the link:
![picture of design](https://user-images.githubusercontent.com/75148774/142981995-5cd2ee24-3b7e-4344-b656-f67d76d347a4.png)
This will not work with links outside of the Scratch website. This could be applied to studios, projects, and users.
Thanks,
webdev03

**Comment by mxmou** [comment]
Related: #1391 and #2271 (both for profile links only, so this isn't a duplicate)

**Reply by imdevarsh** [reply]
Hmm.. thanks!

**Comment by CST1229** [comment]
I think a developer was working on this sometime.

**Reply by WorldLanguages** [reply]
It was Hans I think

**Reply by imdevarsh** [reply]
Well that's nice I'm definitely going to use that addon!

**Reply by cobaltt7** [reply]
Hans died tho

**Reply by Mbrick2** [reply]
> Hans died tho

@RedGuy12 ?????

**Reply by WorldLanguages** [reply]
He is not as active as he used to be.
